src/types/types.ts

```diff
@@ -194,7 +194,7 @@ export interface Asset
 /**
  * Response from asset aggregate endpoint
  */
-export interface AssetAggregate {
+export interface AggregateResponse {
```

Comment:
Unfortunately we cannot just remove it now, since someone might have imported it.
But you can create synonym
`export interface AssetAggregate = AggregateResponse;`

and typedoc needs to be fixed
> Response from asset aggregate endpoint

not valid anymore

Reply:
also you need to do the same with
`EventAggregate` in that case

Reply:
In hindsight, we should have one for each resource type. The API might diverge in the future.

Reply:
that was an initial thought, yes.